injector/dns_disruption_test.go

```diff
@@ -97,16 +100,33 @@ var _ = Describe("Failure", func() {
 
 		Context("disruption is node-level", func() {
 			It("creates node-level iptable filter rules", func() {
-				iptables.AssertCalled(GinkgoT(), "Intercept", "udp", "53", "", "10.0.0.2")
+				iptables.AssertCalled(GinkgoT(), "Intercept", "udp", "53", "", "", "10.0.0.2")
```

Comment:
What do you think if we assert the number of calls to be sure it is called once and not multiple time?
_This is valid for all Assert bellow._

Reply:
We should do it for all calls then. It's not a bad idea but I'd suggest to do that in a separate PR for injector tests.